applications/gfs_init.py

```diff
@@ -0,0 +1,36 @@
+from credit.nwp import build_GFS_init, format_datetime
+from credit import metadata
+import yaml
+import argparse
+import xarray as xr
+import os
+from os.path import join
+import pandas as pd
+
+username = os.environ.get('USER')
+base_path = '/glade/work/username/'.replace("username", username)
+base_out_path = '/glade/derecho/scratch/username/CREDIT/GFS_INIT/'.replace("username", username)
+os.makedirs(base_out_path, exist_ok=True)
+parser = argparse.ArgumentParser()
+parser.add_argument("-c", "--config", help="Path to config file")
+args = parser.parse_args()
+with open(args.config) as config_file:
+    config = yaml.safe_load(config_file)
+
+credit_grid = xr.open_dataset(join(base_path,'miles-credit/credit/metadata/ERA5_Lev_Info.nc'))
```

Comment:
If you need to automatically figure out a path relative to a given file, there is a way to do it with the os library. I use it [here in test_interp.py](https://github.com/NCAR/miles-credit/blob/507c9834a54e9daa252e6af4f559f3032fc9d170/tests/test_interp.py#L8C5-L8C62)